src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,74 @@
  */
 
 public class Plane {
+    private final String type;
+    private final String producer;
+    private final String model;
+    private final int seatsNumber;
+    private final int productionYear;
 
+    public Plane(PlaneBuilder builder) {
```

Comment:
```suggestion
    private Plane(PlaneBuilder builder) {
```
explain why in the comment below